src/vsr/clock.zig

```diff
@@ -368,6 +368,27 @@ pub fn ClockType(comptime Time: type) type {
             }
         }
 
+        pub fn round_trip_time_median_ns(self: *const Clock) ?u64 {
+            // +1 to allow for the standby.
+            var one_way_delays = stdx.BoundedArrayType(u64, constants.replicas_max + 1){};
+            for (self.window.sources, 0..) |source, replica_index| {
+                if (self.replica != replica_index) {
+                    if (source) |sampled| {
+                        one_way_delays.append_assume_capacity(sampled.one_way_delay);
+                    }
+                }
+            }
+
+            if (one_way_delays.count() == 0) {
```

Comment:
maybe

```suggestion
            if (one_way_delays.count() < self.quorum) {
```

to prevent rtt from jerking abruptly when we switch the window?